### DISCUSSION TITLE
Project ideas for 2024 : OpenVINO Extension for Automatic1111 Stable Diffusion WebUI

### DISCUSSION THREAD

**Discussion by Aryan8912** [openvinotoolkit/openvino#22701] [Google Summer of Code]
As part of an ideas list for GSOC 2024, I would like to work with Gradio library specifically designed for Stable Diffusion.

**Comment by adrianboguszewski** [comment]
Thanks for being interested. You will need to fill out our template from https://github.com/openvinotoolkit/openvino/wiki/Google-Summer-Of-Code and submit it in the GSoC web app. But, of course, we need to wait until the official Google announcement.

**Reply by Aryan8912** [reply]
hey sir, I watched your video on YouTube and checked the openvino notebook repository also my Pull Request is under review, one or two days my pull request was accepted.

**Comment by Aryan8912** [comment]
@adrianboguszewski Hey sir what is the next step I have to do

**Reply by adrianboguszewski** [reply]
Please follow the instructions here: https://github.com/openvinotoolkit/openvino/wiki/Google-Summer-Of-Code
